### DISCUSSION TITLE
What is Auto Gen?

### DISCUSSION THREAD

**Discussion by WhatAUsernameIHave** [Deijin27/RanseiLink#73] [General]
Just seeing a lot of commits regarding it, am curious as to what it entails!

**Comment by Deijin27** [comment]
It is automatically generating much of the code for reading and displaying game data, drive from this config file

https://github.com/Deijin27/RanseiLink/blob/master/RanseiLink.Generation/Models.xml

It was tedious to add new properties and have to update lots of places. This also makes it easier to make infrastructural changes as I can update the just generator rather than having to do every file separately.

**Reply by WhatAUsernameIHave** [reply]
I figured, just saw it a lot so I wanted to make sure!
